### PR TITLE
Minor improvement to the description of try/catch/finally.

### PR DESCRIPTION
**Pull request by carlsmith**
Just made it a bit cleaner.

### CODE REVIEW

File: documentation/index.html.js [hunk 1]
```diff
@@ -874,9 +874,8 @@ Expressions
     <p>
       <span id="try" class="bookmark"></span>
       <b class="header">Try/Catch/Finally</b>
-      Try/catch statements are just about the same as in JavaScript, with a few
-      additions: They work as expressions, and you may omit saving the error in
-      the catch—or the entire catch itself—if you don't need it.
+      Try-expressions have the same semantics as try-statements in JavaScript,
+      though in CoffeeScript, the catch and finally parts are both optional.
```
Comment (michaelficarra) [inline documentation/index.html.js]: Technically in JavaScript, they're also both (individually) optional. What you need to say is that they may both be omitted at the same time. Also, it's nice to mention that you can have a catch without the catch parameter.

Comment (lydell) [inline documentation/index.html.js]: Ahh, I had no idea that `try{1}finally{2}` was valid JS.

**Comment by carlsmith** [comment]
Nor I. Gimme ten.

**Comment by carlsmith** [comment]
Opps, used Markdown, when it's HTML.

**Comment by carlsmith** [comment]
F**ing git. Sorry. I was trying to merge the three commits into one, but it didn't work.

**Comment by lydell** [comment]
`git rebase -i 221683f^`. Squash all. `git push -f`.

**Comment by carlsmith** [comment]
It was the `-f` part  that was confusing me. Origin was ahead.

**Comment by carlsmith** [comment]
I did it again, it's now #3974.

**Comment by lydell** [comment]
The point of `git push -f` is _not_ having to create a new PR …

**Comment by carlsmith** [comment]
Yep, sorry. I'd already basically done it though.

If git was less likely to embarrass its user, more people would contribute
to open source.

**Comment by lydell** [comment]
This was superseded by #3974, which was merged, right? So this one can be closed?

**Pull request closed without merging**